src/Microsoft.ML.StandardLearners/Standard/PoissonRegression/PoissonRegression.cs

```diff
@@ -32,7 +32,7 @@ public sealed class PoissonRegression : LbfgsTrainerBase<Float, PoissonRegressio
         internal const string UserNameValue = "Poisson Regression";
         internal const string ShortName = "PR";
         internal const string Summary = "Poisson Regression assumes the unknown function, denoted Y has a Poisson distribution.";
-        new internal const string Remarks = @"<remarks>
+        internal new const string Remarks = @"<remarks>
```

Comment:
@Ivanidzo4ka, It has be reverted here as well...:)
I have opened PR (486) to fix it.